.changeset/orange-kangaroos-greet.md

```diff
@@ -0,0 +1,19 @@
+---
+"@comet/admin": minor
+---
+
+Support applying props directly to `CrudMoreActionsMenu` to apply them to its `Button` component
+
+For example, the component can be set to be responsive by applying the `responsive` prop from `Button`:
+
+```tsx
+<CrudMoreActionsMenu
+    responsive
```

Comment:
Shouldn't you use slotProps for this? 🤔

Reply:
Here is the alternative using `slotProps`: https://github.com/vivid-planet/comet/pull/3236

Reply:
> I think it makes sense that the root-level props are passed directly into the "primary thing" of the component...

I'm not sure about this. For instance, I would then have `onClick` in the props, which will never work, because we overwrite it in the component. Also, we could argue that the "primary thing" in this component is the menu, not the more actions button.

We can either:
1. Use slotProps
2. Add specific props from Button to CrudMoreActionsMenu, for instance, responsive. We should improve the naming though, because only the button becomes responsive, not the menu.

Reply:
You're probably right, maybe we should go with the alternative then 🙂